ush/inter_flux.sh

```diff
@@ -45,9 +45,16 @@ else
 fi
 
 #---------------------------------------------------------------
-  $WGRIB2 $PGBOUT $option1 $option21 $option22 $option23 $option24 \
+
+  if [ $INLINE_POST = ".false." ]; then
+    $WGRIB2 $PGBOUT $option1 $option21 $option22 $option23 $option24 \
+                          $option25 $option26 $option27 $option28 \
```

Comment:
Indentation for line continuations should be consistent with each other

Reply:
should be fixed now